core/src/test/java/org/infinispan/expiration/impl/ExpirationStoreListenerFunctionalTest.java

```diff
@@ -84,7 +84,7 @@ public void testExpirationOfStoreWhenDataNotInMemory() throws Exception {
       assertFalse(event.isPre());
       assertNotNull(event.getKey());
       // The dummy store produces value and metadata so lets make sure
-      if (TestingUtil.getCacheLoader(cache) instanceof DummyInMemoryStore) {
+      if (TestingUtil.getFirstStore(cache) instanceof DummyInMemoryStore) {
```

Comment:
isn't this always true?

Reply:
No, this is extended and sometimes it is SingleFileStore.